### PR TITLE
ZK-4398: Provide PortalChildren title and frame design

### PR DESCRIPTION
**Pull request by plijyfes**

### CODE REVIEW

File: zul/src/archive/metainfo/xml/zul.xsd [hunk 1]
```diff
@@ -2614,6 +2614,8 @@
 			<xs:element ref="panel" />
 		</xs:choice>
 		<xs:attributeGroup ref="xulElementAttrGroup" />
+		<xs:attribute name="title" type="xs:string" use="optional" />
+		<xs:attribute name="childrenCounterVisible" type="booleanType" use="optional" />
```
Comment (scribetw) [inline zul/src/archive/metainfo/xml/zul.xsd]: counterVisible

File: zul/src/archive/web/js/zul/wnd/less/panel.less [hunk 2]
```diff
@@ -35,6 +35,10 @@
 			cursor: move;
 		}
 	}
+
+	&-dragbtn {
```
Comment (scribetw) [inline zul/src/archive/web/js/zul/wnd/less/panel.less]: &-drag-button

File: zul/src/archive/web/js/zul/wnd/mold/panel.js [hunk 3]
```diff
@@ -57,7 +57,9 @@ function (out, skipper) {
 			out.push(zUtl.encodeXML(title));
 		}
 		out.push('</div></div>');
-	} 
+	} else {
+		out.push('<div id="', uuid, '-dragbtn" class="', this.$s('dragbtn'), " z-icon-minus", '"></div>');
```
Comment (scribetw) [inline zul/src/archive/web/js/zul/wnd/mold/panel.js]: should be designed

**Comment by scribetw** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I could not rebase F90-ZK-4398 with master. There are conflicts.

**Pull request merged**